internal/controller/scwexternalip_controller.go

```diff
@@ -54,16 +57,27 @@ var (
 	finalizer    = "ptrk.io/finalizer"
 )
 
+const (
+	cacheKey string = "scwIpCache"
+)
+
 type NodeNameID struct {
-	Name string
-	ID   string
+	Name   string
+	ID     string
+	EIpIds []string
+}
+
+type Cache interface {
+	Set(cacheKey string, value interface{}, maxAge time.Duration)
```

Comment:
can we use the `*instance.ListIPsResponse` type instead since we only use this type? Or use generics :smile: